packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -29,6 +28,13 @@ export class GaussianSplattingMesh extends Mesh {
     private _centersTexture: Nullable<BaseTexture> = null;
     private _colorsTexture: Nullable<BaseTexture> = null;
     private _splatPositions: Nullable<Float32Array> = null;
+    //@ts-ignore

```

Comment:
yes, it's meant to be used in the (close?) future by an editor and various processes.

Reply:
just wondering - why is it needed actually?

Reply:
There is a build error because these members are not used (yet).

Reply:
In that case, `@ts-expect-error` is probably a little better, because you can't forget to remove it when you do start using the member. `ts-expect-error` will itself cause a build error if the next line *does not* produce a build error.

Reply:
I like that! Fixed!